filament/include/filament/Engine.h

```diff
@@ -335,6 +335,14 @@ class UTILS_PUBLIC Engine {
         destroy(camera->getEntity());
     }
 
+   /**
+     * Invokes one iteration of the render loop, used only on single-threaded platforms.
+     * 
+     * Note that the OpenGL context that was created during Engine construction needs to be
+     * current at the time that this is called.
+     */
+    void tick() noexcept;
```

Comment:
The comment is kind of meaningless in a public header since we build the GL context and we make it current. Also we don't necessarily have a GL backend :)